src/components/InteriorLeftNavList/InteriorLeftNavList.js

```diff
@@ -105,7 +105,7 @@ export default class InteriorLeftNavList extends Component {
         <ul
           role="menu"
           className="left-nav-list left-nav-list--nested"
-          aria-hidden>
```

Comment:
I'm a bit concerned about removing this as it's making sure that the content is not being read by a screen reader when it's hidden (at least that's my understanding).